docs/go/signals.md

```diff
@@ -27,13 +27,13 @@ To uniquely identify the Workflow, we need to pass in the `workflowID` and `runI
 Typically, we signal a Workflow from a different process, like a [starter](/docs/go/hello-world-tutorial/#workflow-starter).
 
 ```go
-temporal, err := client.NewClient(client.Options{})
+c, err := client.NewClient(client.Options{})
```

Comment:
Something a bit more descriptive and precise might be `temporalClient` - what do you think about that?